pkg/util/flowexport/flowexport.go

```diff
@@ -33,7 +33,7 @@ func ParseFlowCollectorAddr(addr string, defaultPort string, defaultProtocol str
 	}
 	if match {
 		idx := strings.Index(addr, "]")
-		strSlice = append(strSlice, addr[:idx+1])
+		strSlice = append(strSlice, addr[1:idx])
```

Comment:
was this a bug? Previously the same is working when flow exporter is supported for IPv6?

Reply:
Yes, I found this bug when I was connecting a IPv6 ipfix collector to the flow exporter directly.

Reply:
Wonder how standalone flow exporter e2e tests with external flow collector were passing before? Requesting review from Zhecheng who worked on Supporting IPv6 in Flow Exporter.

Reply:
There are some changes between my PR and this one. You can see that `host` and `port` were combined with a `:` directly. Then it worked well if `host` is returned as `[<ipv6-addr>]`.
https://github.com/vmware-tanzu/antrea/pull/1718/files#diff-881e8cbabcf6cfcfec09e0ad632f103993d06a26cdf31d22d30cdbed14b14597L96